yarn-project/noir-contracts/src/contracts/card_game_contract/src/game.nr

```diff
@@ -31,36 +31,25 @@ struct Game {
 global GAME_SERIALIZED_LEN: Field = 15;
 
 fn deserializeGame(fields: [Field; GAME_SERIALIZED_LEN]) -> Game {
-    let players = [
-        PlayerEntry {
-            address: fields[0],
-            deck_strength: fields[1] as u32,
-            points: fields[2] as u120,
-        },
-        PlayerEntry {
-            address: fields[3],
-            deck_strength: fields[4] as u32,
-            points: fields[5] as u120,
-        },
-    ];
-    let rounds_cards = [
-        Card::from_field(fields[6]), Card::from_field(fields[7]),
-        Card::from_field(fields[8]), Card::from_field(fields[9]),
-    ];
-    Game { 
+    let players = [PlayerEntry { address: fields[0], deck_strength: fields[1] as u32, points: fields[2] as u120 },
+        PlayerEntry { address: fields[3], deck_strength: fields[4] as u32, points: fields[5] as u120 }];
```

Comment:
can we align PlayerEntry to be below L34's playerEntry? Hurts eyes hahha

FWIW I prefer the original

Reply:
New nargo version no longer did this change. Once again great job @f01dab1e